visualization/app/codeCharta/ui/customConfigs/customConfigs.component.ts

```diff
@@ -53,11 +56,18 @@ export class CustomConfigsController implements FilesSelectionSubscriber {
 
 	onFilesSelectionChanged(files: FileState[]) {
 		this.customConfigFileStateConnector = new CustomConfigFileStateConnector(files)
+
+		this.prefetchDownloadableConfigsForUploadedMaps()
+	}
+
+	initView() {
+		this.loadCustomConfigs()
+		this.prefetchDownloadableConfigsForUploadedMaps()
```

Comment:
I think I am confused by the name _prefetchDownloadableConfigsForUploadedMaps_. Would something like `updateApplicableConfigs` be a better description? I mean there is nothing fetched / downloaded but only `this.downloadableConfigs` recalculated by already read value of customConfigs from local storage, right?

Reply:
`this.downloadableConfigs` must be recaluclated because the user can switch the selected map and add Custom Configs at any time. I like your idea to rename the function. I will shorten the name because it likely confuses developers and replace prefetch, with preload.